flask_potion/contrib/alchemy/manager.py

```diff
@@ -86,6 +87,14 @@ def _init_model(self, resource, model, meta):
                     fs.required.add(name)
                 fs.set(name, self._get_field_from_column_type(column, name, io=io))
 
+    def _get_sort_expression(self, model, meta, id_column):
+        if meta.sort_attribute is None:
+            return id_column.asc()
```

Comment:
Does it sort everything by default on primary key column ?
What if we do not want to have this behaviour ?
Sorting can have substantial impact on performance for instance.

Reply:
That has been the behavior for a while. Without an intrinsic sort order, it is not possible to do reliable pagination. Since the primary key is indexed, it is a good default. Do you see a practical scenario where no sort is preferred? We could support `sort_attribute=False` to mean no sorting.